data/monster/lua/test.lua

```diff
@@ -0,0 +1,50 @@
+-- a testing layout
+local test = NewMonsterType('Test')
```

Comment:
Evil, to compile here I had to change the cmake to compile. Travis is
alright.

On Sun, Apr 19, 2015, 13:39 Mateusz Pawlica <notifications@github.com>
wrote:

> In data/monster/lua/test.lua
> <https://github.com/otland/forgottenserver/pull/1296#discussion_r28654416>
> :
>
> > @@ -0,0 +1,50 @@
> > +-- a testing layout
> > +local test = NewMonsterType('Test')
>
> Keep it simple, insetad NewMonsterType you can do something like this:
>
> local test = MonsterType('Test')
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/otland/forgottenserver/pull/1296/files#r28654416>.
>